mmv1/products/billingbudget/terraform.yaml

```diff
@@ -93,8 +93,6 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         custom_flatten: 'templates/terraform/custom_flatten/object_to_bool.go.erb'
       amount.specifiedAmount.currencyCode: !ruby/object:Overrides::Terraform::PropertyOverride
         default_from_api: true
-      budgetFilter.creditTypes: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Do these overrides need to be removed to make the field updatable?

Reply:
Without removing the overrides, the test failed. https://github.com/GoogleCloudPlatform/magic-modules/pull/7035/files#diff-a5c0fe03fa3b54255bdca47bd1415a24c2de3ec1715a457db27a558bd0e1b530R114. 

The error is "Credit types treatment of INCLUDE_ALL_CREDITS or EXCLUDE_ALL_CREDITS requires empty list of credit_types".

The reason is that creditTypes is sent to API because of the `computed` override even the user does not set that field in config.

Reply:
I checked the response from API and the API does not return default values for the two fields. These two fields do not behave like computed fields.

Reply:
This should be able to be fixed while retaining the computed property. Even though this is a mistake these changes will break end users.

Reply:
@ScottSuarez , do you have an idea how to fix it while retaining the computed property?